docs/source/using/profiles.adoc

```diff
@@ -8,18 +8,45 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state information.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+Profiles share nothing between them. Things like cache, add-ons and persistent config are specific to a single profile.
+This means if you have set some specific configuration parameters for a profile, these settings do not apply to other profiles.
+
+Currently, we cannot set any global configuration for profiles.
```

Comment:
Here I am losing the red thread again. I think this whole discussion should be moved into a dedicated TIP or section.

Reply:
This information is important enough that I think a section is appropriate.

Reply:
Right, here is probably a good point to leave the overview. The overview should define the feature and give some context. That we have done now. The rest should be in dedicated sections, preferably in some logical order.